ValidModeSetup.xml

```diff
@@ -0,0 +1,261 @@
+<?xml version="1.0" encoding="utf-8" standalone="no" ?>
+<!--
+	-Mode is declared in order, so from 1(GrainTransportAIDriver) to 10(LevelCompactAIDriver)
+	-do not change the order here!
+	
+	-allowedSpecializations is used to check:
+		-if one of the <Setup> combo is true for each object
```

Comment:
However, it needs a better explanation and probably better XML tags.
- explain what 'object' is and what 'each object' refers to
- I think 'allowedSetups' and 'disallowedSetups' would be clearer.

for example (not sure if this is correct, rewrite if you want)

"A mode is enabled if the specializations of the vehicle and its attached implements and trailers:
* match one of the Setups under allowedSetups, and,
* do not match any of the Setups under the disallowedSetups

A Setup is a combination of one or more specializations. To match a Setup, the vehicle must have all Specializations listed in the Setup."

Reply:
Or maybe just 'allow' 'disallow' BTW, why lowercase, everything else is uppercase?